app/install/source.md

```diff
@@ -36,6 +34,8 @@ redirect_from: /install/compile/
     $ ./configure --with-pcre-jit --with-ipv6 --with-http_realip_module --with-http_ssl_module --with-http_stub_status_module
     ```
 
+    Install [Luarocks v{{luarocks_version}}](https://github.com/keplerproject/luarocks/wiki/Download) (note: configure to run with the Lua/LuaJit interpreter included with OpenResty/Nginx)
```

Comment:
Best to say that LuaJIT 2.1 is needed (in the $PATH), and that when compiling OpenResty, it is possible to point to that LuaJIT. Same when compiling Luarocks.